### ISSUE TITLE
Can't see/find epub, mobi, pdf versions

### ISSUE THREAD

**Issue by walkjivefly** [ProgrammingBlockchain/ProgrammingBlockchain#121]
There is a link in the links section of [https://programmingblockchain.gitbook.io/programmingblockchain/](url) which says "you can download pdf, epub and mobi versions here."
Except you can't. It's a circular link back to the same page.

Even signed in to Gitbooks I can't see anywhere to download alternate formats. Am I just not seeing it, or does it not exist?

**Comment by nopara73** [comment]
GitBook changed some stuff. Not sure what exactly, but that link was completely different back then.

**Comment by NicolasDorier** [comment]
yeah I updated the gitbook recently as they stopped synching with github because of this.
I am trying to find where to export...

**Comment by NicolasDorier** [comment]
The default link https://programmingblockchain.gitbook.io goes to indonesian... they fucked it up

**Comment by NicolasDorier** [comment]
![image](https://user-images.githubusercontent.com/3020646/47066951-aee89800-d222-11e8-84a0-230024385001.png)

**Comment by NicolasDorier** [comment]
I set back the default link to english

**Comment by nopara73** [comment]
I removed the outdated "you can check pdf here" note https://github.com/ProgrammingBlockchain/ProgrammingBlockchain/commit/636858ddea557b972c0642004d07f52e15bdf986

By the way, I'm happy it doesn't export to pdf anymore, because they weren't doing it properly anyway. When I tried the .pdf version much of the code was clipped and stuff like that.

**Comment by walkjivefly** [comment]
OK, so no offline versions available. 
That's a shame, I wanted to be able to refer to the manual on my Kindle. Not having a multi-monitor setup or a vast single monitor it's a PITA having to switch between windows rather than seeing stuff side-by-side.

**Comment by NicolasDorier** [comment]
I agree this sucks :/

**Comment by nopara73** [comment]
Yeah, it would be worth the effort to go through it again, then release it as a normal book, unfortunately we are way too busy coding now.

**Comment by gked** [comment]
also, there is currently no way for me to leave comments in the gitbook. I have tried registering with gitbook but it redirects me their new site. Has anyone else experienced problems with it?
